docs/workflow.md

```diff
@@ -21,6 +21,24 @@ This repo has bots that manage all stale PRs. Stale PRs will be autoclosed.
 - *Do* use GitHub's tooling. Re-request review after all feedback has been addressed.
 - *Do* pay special attention to the commit message. Ensure the merge message is appropriate and helpful to the future reader. See [merge commit considerations](#merge-commit-considerations).
 
+#### Pull Request Etiquette and Best Practices
+
+- If you disagree with the overall approach of the PR, comment on the general PR rather than individual lines of code.
```

Comment:
On GitHub, I disagree.

If you comment on a line of code, it is a github" conversation" where posts/comments are grouped together, and the whole thread (github calls it a conversation) can be marked as resolved. On the other hand, if you write a comment at the bottom of the "conversation" view on the PR, it's not threaded (comments grouped together) and can't be marked as resolved.

Something you can do is use the "comment on file" feature, so you still have to pick a file to comment on, but it won't be a comment on a specific line of code.